src/index.d.ts

```diff
@@ -88,6 +90,17 @@ export function createRoutine<Payload, Meta, Arg1, Arg2, Arg3, Arg4>(
     metaCreator: ActionFunction4<Arg1, Arg2, Arg3, Arg4, Meta>
 ): Routine<ActionFunction4<Arg1, Arg2, Arg3, Arg4, ActionMeta<Payload, Meta>>>;
 
-export function bindRoutineToReduxForm<T>(routine: Routine<T>): FormSubmitHandler;
+export interface ReduxFormPayload<FormData = {}, P = {}> {
+    values: FormData
+    props: P
+}
+
+export function bindRoutineToReduxForm<Payload extends ReduxFormPayload<FormData, P>>(
```

Comment:
`FormData` here refers to global `FormData` object which is totally unrelated to `redux-form`'s type.

Reply:
True, somehow I've lost part of the PR - #45